content/docs/user-guide/components/reference/terrain/world-renderer.md

```diff
@@ -11,6 +11,20 @@ The **Terrain World Renderer** component renders the terrain within the bounds s
 You must add this component to the *Level* entity, the parent of all entities in an **Open 3D Engine (O3DE)** level.
 {{< /important >}}
 
+## Usage 
+The terrain renderer component allows terrain to be visualized in the world and controls various global properties that apply to terrain everywhere. 
+
+#### Mesh Configuration
+These properties control the mesh density and blending of terrain LODs (level of detail) and have a very direct impact on vertex performance. The density of the first terrain
+LOD is set by the query resolution in the [Terrain World](/docs/user-guide/components/reference/terrain/world) component, and **First LOD distance** controls how far to render at this
+density. Each subsequent LOD will render for twice the distance at half the resolution all the way until **Mesh render distance** is met. This means that **First LOD distance** actually
+has a much greater impact on total triangle count than the total rendering distance.
+#### Detail Configuration
+These settings impact how far detail materials render, how they fade out, and how they blend with each other. Most of these settings have little impact on performance, however a larger
+render distance here will require more surface data points to be queried as you move around the world and also take more memory.
+#### Clipmap Configuration
+Turning on clipmaps causes all the material blending to be baked into textures for a substantial gain in performance, but also comes with a substantial increase in memory cost.
```

Comment:
From what I've seen so far, I'm not sure it's true that we can say there's a substantial gain in performance. Steady-state rendering performance appears to be faster, but if you fly rapidly around the world, the clipmaps currently seem to have far worse performance than the forward rendering version.

Reply:
Suggestion: This could also benefit from a brief description of what clipmaps are and how they work. I believe Jiawei posted a helpful diagram on a PR showing what's happening here.

Reply:
I was intentionally light on info with clip maps since I don't know all the details of its implementation. It'd be great for that to be fleshed out by @jiaweig-amzn :). As for the statement about performance, clipmaps _should_ be substantially faster in most normal applications. I'm sure there are many opportunities to make them more performant as time goes on, but maybe we need to be more precise about their expected performance in different applications.

Reply:
I have sketched this out. Feel free to modify any word. The sentence below in brackets is probably too much detail. We can remove that.
Clipmaps are a stack of baked textures centered at the view and covering different distances at different resolutions. Detail materials and macro materials are backed into 2 sets of clipmaps which will take their place when the terrain is rendered. (In O3DE implementation, clipmaps have a uniform size, and the least index stands for the highest resolution clipmap.) Using clipmap will have a substantial gain in performance at the cost of texture memories. Clipmap configuration can be used to find a balance between the performance gain and the memory cost.